// Error: x is declared twice in the same scope
package main

func main () {
	x := 0
	var x int
}